package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@colony/eslint-config-colony",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
I think this should be done in a separate commit `npm version patch`

Reply:
Agreed, let's do it on master